src/components/Trade/hooks/useFiatRateService.tsx

```diff
@@ -43,14 +47,13 @@ export const useFiatRateService = () => {
 
   // Trigger fiat rate query
   useEffect(() => {
-    if (sellTradeAssetId && buyTradeAssetId && sellAssetFeeAssetId) {
+    if (sellTradeAssetId && buyTradeAssetId && sellAssetFeeAssetId && tradeQuoteArgs) {
       setUsdRatesArgs({
-        buyAssetId: buyTradeAssetId,
-        sellAssetId: sellTradeAssetId,
+        tradeQuoteArgs,
```

Comment:
curious about this API change, buyAssetId and sellAssetId seemed cleaner. is this the bit where we required the full asset?

Reply:
TL;DR; USD rates are swapper-specific. Which swapper do we use to get rates? We need to get a trade quote first to work it out.

---

Ah yes, this was a second order effect of implementing best swapper. 

We require a swapper to get USD rates. Previously, we'd get the (best) swapper using only buy and sell asset IDs as we'd just get the first one that supported the pair.

Now, to get a (best) swapper we need to provide sufficient args to get a quote, as a quote is now required to work out which swapper to use.